docs/docs/Core/map.md

```diff
@@ -0,0 +1,89 @@
+---
+id: map
+title: Map
+---
+
+A `Map` is a dictionary of key/value pairs, in which you can get a value associated with its key.
+
+
+```dart
+Map map = {
+    "content": "This is a message"
+};
+
+screm(map["content"]);
+```
+
+## Properties
+
+### isEmpty
+
+`bool isEmpty`
+
+True is this Map is empty
+
+**Example**
+```dart
+Map map = {
+    "content": "This is a message"
+};
+bool isEmpty = map.isEmpty; // false
+```
+
+### isNotEmpty
+
+`bool isNotEmpty`
+
+True if this Map is not empty.
```

Comment:
Specify that it returns a bool for consistency. `Returns a bool that is true if this Map is not empty.` do the same for isEmpty pls